app/validators/WorkerProject.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace GanttDashboard\App\Validators;
+
+use Phalcon\Validation\Validator\PresenceOf;
+use Phalcon\Validation\Message;
+use GanttDashboard\App\Services\Project as ProjectService;
+
+
+class WorkerProject extends Base
+{
+    /**
+     * Constructs the validations
+     */
+    public function __construct()
+    {
+        $this->add(
+            'submit',
+            new PresenceOf([
+                'cancelOnFail' => true,
+            ])
+        );
+    }
+
+    /**
+     * Executed before validation
+     *
+     * @param array $data
+     * @param object $entity
```

Comment:
The type must be one of the following: 

- a class name, either Fully Qualified or aliases; such as \DateTime or Entity (with a namespace alias elsewhere in the same source file).
- a keyword for a primitive in PHP, such as int or string.
- a special keyword specific to the PHPDoc Standard, such as false or mixed.

"object" is not accepted